Makefile.third

```diff
@@ -317,6 +317,25 @@ ifdef ANDROID
 	readelf -d $@ | grep "no dynamic" \
 		|| echo "warning: Android L+ support only PIE binary and can't run this"
 endif
+# ===========================================================================
+# dropbear
+
+$(OUTPUT_DIR)/dropbear: $(THIRDPARTY_DIR)/dropbear/CMakeLists.txt
+	install -d $(DROPBEAR_BUILD_DIR)
+	cd $(DROPBEAR_BUILD_DIR) && \
+		$(CMAKE) -DCC="$(CC)" \
+		-DLIBS="$(if $(or $(ANDROID),$(WIN32),$(DARWIN)),,-lrt) $(if $(ANDROID),-static,)" \
+		$(if $(LEGACY),-DDISABLE_LARGEFILE:BOOL=ON -DDISABLE_FORTIFY:BOOL=ON,) \
```

Comment:
What's this LEGACY stuff exactly? Is it specifically for the Kindle 4NT (and similarly old devices)?

Reply:
I'm investating and I think it's for device <= kindle3 and pocketbook